package.json

```diff
@@ -7,14 +7,19 @@
     "moon": "moon",
     "pre-commit": "lint-staged",
     "prepare": "husky install",
+    "prettier": "npm run prettier:code &&npm run prettier:ui",
+    "prettier:ui": "npx prettier --check --config .prettierrc \"docs/ui/src/**/*.*\"",
+    "prettier:code": "npx prettier --check --config .prettierrc \"packages/contrast-colors/lib/**/*.*\"",
+    "prettier:fix": "npm run prettier:code -- --write && npm run prettier:ui -- --write",
```

Comment:
I think these are unnecessary. We should use prettier on everything that isn't excluded by the `.prettierignore` and defined in the [.lintstagedrc.json](https://github.com/adobe/leonardo/blob/monorepo/.lintstagedrc.json) file.

Reply:
@GarthDB Thank you!! yes, I saw the `.lintstagedrc.json` entry, but I would like to be able to run `prettier` without doing a commit. Sorry, I am new to monorepo and not sure, if/where tools like prettier would be added to the monorepo? I saw this: https://moonrepo.dev/docs/guides/examples/prettier#setup
I will remove the npm script entries, but shouldn't I add a `.moon/tasks/node.yml`?

Reply:
A node.yaml tasks file for type checking? Or are there other project wide node scripts you think should be in there?

Reply:
@GarthDB Well, after our Slack chat, I think, it's okay to configure the IDE to run prettier on "file save". So, no need for that node.yaml at this point. But still curios, if that  `.moon/tasks/node.yml` is a good pattern for moon mono repos...